src/api/rtd_csv_transaction/create-sas-token-policy.xml.tpl

```diff
@@ -0,0 +1,268 @@
+            }"
+        />
+        
+        <set-variable name="containerName" value="@{
+            return string.Format("{0}-{1}",
+                (string)context.Variables["containerPrefix"],
+                ((string)context.Variables["apimSubscriptionKeyHash"]).Substring(0, 44));
+            }"
+        />
+        <!--
+            END - Derive the client's own container from the APIM Subscription Key
+        -->
+
+        <!--
+            START - Create a Shared Key to authorize APIM to create a container into a storage account
+
+            see: https://docs.microsoft.com/en-us/rest/api/storageservices/authorize-with-shared-key
+        -->
+        <set-variable name="createContainerDate" value="@(DateTime.UtcNow.ToString("R"))" />
+        <set-variable name="createContainerXmsVersion" value="2020-12-06" />
+
+        <set-variable name="createContainercanonicalizedHeaders" value="@{
+            return string.Format("x-ms-version:{0}",
+                (string)context.Variables["createContainerXmsVersion"]);
+            }"
+        />
+
+        <set-variable name="createContainercanonicalizedResource" value="@{
+            return string.Format("/{0}/{1}\nrestype:container",
+                (string)context.Variables["storageAccount"],
+                (string)context.Variables["containerName"]);
+            }"
+        />
+
+        <set-variable name="StringToSign" value="@{
+            return string.Format(
+                "PUT\n\n\n\n\n\n{0}\n\n\n\n\n\n{1}\n{2}",
+                (string)context.Variables["createContainerDate"],
+                (string)context.Variables["createContainercanonicalizedHeaders"],
+                (string)context.Variables["createContainercanonicalizedResource"]);
+            }"
+        />
+
+        <set-variable name="sharedKey" value="@{
+                // https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/authentication-for-the-azure-storage-services
+                // Hash-based Message Authentication Code (HMAC) using SHA256 hash
+                System.Security.Cryptography.HMACSHA256 hasher = new System.Security.Cryptography.HMACSHA256(Convert.FromBase64String((string)context.Variables["accessKey"]));
+                return Convert.ToBase64String(hasher.ComputeHash(System.Text.Encoding.UTF8.GetBytes((string)context.Variables["StringToSign"])));
+            }"
+        />
+
+        <set-variable name="createContainerAuthorizationHeader" value="@{
+            return string.Format(
+                "SharedKey {0}:{1}",
+                (string)context.Variables["storageAccount"],
+                (string)context.Variables["sharedKey"]);
+            }"
+        />
+        <!--
+            END - Create a Shared Key to authorize APIM to create a container into a storage account
+        -->
+
+        <!--
+            START - Perform a PUT operation on the storage REST endpoint
+        -->
+        <set-variable name="createContainerUrl" value="@{
```

Comment:
The storage should be accessed via a private url (e.g. a private endpoint), maybe we could fix it as a part of another pr